include/volume/volume_cooling_gaussians.hpp

```diff
@@ -204,56 +204,53 @@ void compute_annealing_schedule(Polytope& P,
     get_first_gaussian(P, frac, chebychev_radius, error, a_vals);
 
 #ifdef VOLESTI_DEBUG
-    std::cout<<"first gaussian computed\n"<<std::endl;
+    std::cout << "first gaussian computed\n" << std::endl;
 #endif
 
     NT a_stop = 0.0;
     const NT tol = 0.001;
     unsigned int it = 0;
     unsigned int n = P.dimension();
-    const unsigned int totalSteps = ((int)150/((1.0 - frac) * error))+1;
-
-    if (a_vals[0]<a_stop) a_vals[0] = a_stop;
```

Comment:
Why are you removing this check? If the computed (first) gaussian has negative variance then we set it to zero. @TolisChal is this check actually needed?